src/routes/jira/jira-get.ts

```diff
@@ -194,14 +195,26 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 	});
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const getFailedSyncErrors = async (subscription: Subscription) => {
+	const RETRYABLE_ERROR_CODES = ["PERMISSIONS_ERROR", "CONNECTION_ERROR"];
+	const failedSyncs = await RepoSyncState.getFailedFromSubscription(subscription);
+	const errorCodes = failedSyncs.map(sync => sync.failedCode);
+	const retryableErrorCodes = errorCodes.filter(errorCode => RETRYABLE_ERROR_CODES.includes(errorCode || ""));
```

Comment:
For code readability, in `RETRYABLE_ERROR_CODES.includes(errorCode || "")`, what does `|| "")` means as a param in `.includes()`? I find a a bit hard to wrap my head around.

Reply:
yeah agree, that was not intended originally, it was typescript having a whinge, ill restructure it a bit so that it can drop the || ""